### ISSUE TITLE
[SelectableTile]: VoiceOver announces tiles as group and doesn't read status

### ISSUE THREAD

**Issue by janhassel** [carbon-design-system/carbon#5082]
## Environment

> Operating system
macOS 10.14.6

> Browser
Chrome 79
Safari 12.1.2

> Assistive technology used to verify
VoiceOver

## Detailed description

> What version of the Carbon Design System are you using?
carbon-components-react@7.9.1

When tabbing through a group of SelectableTiles or RadioTiles, VoiceOver does not announce if a tile is selected or not and just reads out "[tile title], group" in Safari and just "group" in Chrome.

## Steps to reproduce the issue

1. Open http://react.carbondesignsystem.com/?path=/story/tile--multi-select
2. Click on first tile
3. Enable VoiceOver
4. Tab through the tiles and select some by pressing space or return

## Additional information

Seems to be very similar to #2484

**Comment by abbeyhrt** [comment]
Hi @janhassel! Thank you for taking the time to make this issue! I believe this has been resolved. If you go to http://carbon-components-react.netlify.com/, you will see the storybook that is tracking the master branch with the changes that will be introduced in new releases. For Multiselect Tiles, you can see here that it is announcing the checked/unchecked state. 
![Multiselect-vo](https://user-images.githubusercontent.com/17281178/72639027-5a066680-392a-11ea-8e1b-98cccad8b026.gif)
For the Radio Tiles, you can see here that VO announces whether or not the tile is selected: 
![Radio-Tiles-VO](https://user-images.githubusercontent.com/17281178/72639394-1d873a80-392b-11ea-9db5-e8a9890e6f27.gif)
Let me know if this is not what you find!

**Comment by janhassel** [comment]
Hey @abbeyhrt, thanks for the quick response and the link!

Maybe I'm confused because I can interact with other Carbon components solely by tabbing though. However, when I want to achieve what I can see in your screen recordings, I need to actively move the VO-Cursor. Looking at the form example though, it works by just pressing "tab" repeatedly.

Could it be that my VO is misconfigured?

**Comment by joshblack** [comment]
@janhassel it may be related to the fact that VO Focus and Browser focus are two separate things. The browser can focus interactive elements (or things labelled with tabindex), while VO and other screen readers are able to focus various parts of the Accessibility Tree, including things like groups, that aren't necessarily interactive widgets on the page.

**Comment by janhassel** [comment]
That makes sense, thanks!

I guess my next question would then be, do we want to make sure the information also gets announced when using regular tab? Personally, I found this confusing because it seemed off-standard compared to the rest of the components, but on the other side I also don't rely on a screenreader and don't know if a visually impaired person would have the same issue.

On another note, in the screen recordings provided by @abbeyhrt it seems like you need to move the VO-Cursor twice to get to the next tile. Is this correct?

**Comment by joshblack** [comment]
@janhassel if I understand correctly, the active element ends up being the `<label>` which is why the checked state doesn't get announced. We might be able to update what is announced, but ultimately would be fighting against the semantics for this element.

In general, I think these tile patterns might need to be changed to a listbox so that it is easier for folks to know if the specific option is selected or not.

cc @dakahn do you think that'd be a relevant use of ARIA to help out here?

**Comment by dakahn** [comment]
Just jumping in on this now. After some subsequent exploration (thanks @joshblack) it seems like telegraphing state changes via an aria live region might be in order. Definitely can open an enhancement ticket for that and get it in the backlog. I'll close this ticket for now since using VO focus what we want read is being read 👍
